### ISSUE TITLE
The number of the dialog is more than 1115+19

### ISSUE THREAD

**Issue by ChineseYjh** [NathanDuran/Switchboard-Corpus#2]
The papers bellow all show that the total number of the dialog is **1115+19**

- A Dual-Attention Hierarchical Recurrent Neural Network for Dialogue Act Classification

- Dialogue Act Classification with Context-Aware Self-Attention

- Dialogue Act Sequence Labeling using Hierarchical encoder with CRF

But this codes generates **21** and **300** dialogs respectively for val set and dev set in addition to the 1115+19 dialogs. So I wonder **whether the 321 dialogs are from the raw SWDA**. This is really confusing.

**Comment by NathanDuran** [comment]
Sorry for the late response. I'm not used to people using my code.
The original training set is 1115 and testing set is 19, as you say. However, there are 21 'unused' dialogues left in the full corpus, so I assigned them to the validation set.

The 300 dialogues in the dev set were all taken from the training set. I intended to use them as a smaller training set for development purposes. I have since removed this set though, because it is confusing and unnecessary.

Hopefully that answers your question.
